script/engine.js

```diff
@@ -681,7 +694,19 @@
 			if(lang && typeof Storage != 'undefined' && localStorage) {
 				localStorage.lang = lang;
 			}
+		},
+
+		setTimeout: function(callback, timeout, skipDouble){
+
+			if( Engine.options.doubleTime && ( typeof skipDouble == "undefined" || skipDouble == false ) ){

```

Comment:
You can replace `typeof skipDouble == "undefined" || skipDouble == false` with just `!skipDouble`. Javascript will take care of all the typing for you.